addons/web/static/src/js/widgets/debug_manager.js

```diff
@@ -473,7 +473,7 @@ DebugManager.include({
                         model: 'ir.default',
                         method: 'set',
                         args: [
-                            self._controller.fields_view.model,
+                            self._controller.fields_view ? self._controller.fields_view.model : self._action.res_model,
```

Comment:
Not sure at all, but consequently of the action manager refactoring rev. 1 (https://github.com/odoo/odoo/commit/32b8cec5361174c92c45a4975d4e02f1fb448aa2#diff-70ddda003747cd9d01116f29fe79c084L108), I think the equivalent info may be retrieved with this:

```
_.findWhere(self._action.views, {
   type: self._controller.viewType
}).fieldsView.model
```